frontend/apps/marketing/src/components/contentful/activityCatalog/activitiesHero.tsx

```diff
@@ -0,0 +1,114 @@
+'use client';
+
+import {Box, Button, Stack, Typography} from '@mui/material';
+import Link from 'next/link';
+import {usePathname} from 'next/navigation';
+
+export default function ActivitiesHero() {
+  const pathname = (usePathname() ?? '').toLowerCase();
+
+  // Where the buttons navigate to
+  const AI_HREF = '/activities/hour-of-ai';
+  const LEGACY_HREF = '/activities/hour-of-code';
```

Comment:
```suggestion
  const HOUR_OF_AI_PATH = '/activities/hour-of-ai';
  const HOUR_OF_CODE_PATH = '/activities/hour-of-code';
```

Reply:
For consistency, you may want to just inline the hrefs. The other places in the code seem to use inlines.